### ISSUE TITLE
Problem with PET image plot in brainweb

### ISSUE THREAD

**Issue by rajat-mae** [casperdcl/brainweb#1]
Hi Casper,

When I'm trying to run brainweb, as PET output I am getting a blank plot.

Please see the screenshot as attached.

Could you please assist to fix it. I tried running the code many times.

![Screenshot_20200219-194043_Gallery](https://user-images.githubusercontent.com/31813411/74822269-975a7b80-5350-11ea-9c3b-084987686abc.jpg)

**Comment by casperdcl** [comment]
Are you using the code straight from the [example](https://github.com/casperdcl/brainweb/blob/master/README.ipynb) without any alteration?

If so could you post the output of:

```python
import brainweb, sys, os
print(brainweb.__version__, sys.version, sys.platform, os.name)
```

**Comment by rajat-mae** [comment]
> Are you using the code straight from the [example](https://github.com/casperdcl/brainweb/blob/master/README.ipynb) without any alteration?
> 
> If so could you post the output of:
> 
> ```python
> import brainweb, sys, os
> print(brainweb.__version__, sys.version, sys.platform, os.name)
> ```

Yeah, I am trying the example as it tis visualize the tool first.

The output of the above code is:
1.5.0 3.7.4 (default, Aug  9 2019, 18:34:13) [MSC v.1915 64 bit (AMD64)] win32 nt

**Comment by casperdcl** [comment]
@rajat-mae is it possibel that cmaps are somehow broken on your system? Could you change `hot` and `bone` to `Greys_r` and see if that fixes the PET & uMap display?

**Comment by rajat-mae** [comment]
Its still the same, I tried this thing earlier too.
![Untitled](https://user-images.githubusercontent.com/31813411/76283775-430b4180-62e7-11ea-9c32-9daf9edb6008.png)

**Comment by casperdcl** [comment]
This is odd. What's the output of `print(vol['PET'].mean())` and `print(vol['PET'].std())`? Zero?

**Comment by rajat-mae** [comment]
yeah, its coming as zero.

**Comment by casperdcl** [comment]
fixed in `brainweb>=1.5.1`
